NEWS.md

```diff
@@ -0,0 +1,276 @@
+# News
+
+## The great Traversal removal (June 2023)
+### Summary
+We remove the `overflowdb.traversa.Traversal` class, and replace it with a type alias `type Traversal[+T] = scala.collection.Iterator[T]`.
```

Comment:
Can we write this paragraph (and potentially others) in past tense, since the target audience will read it after the PR got merged..?